### DISCUSSION TITLE
KeyValuePairs overflow

### DISCUSSION THREAD

**Discussion by eliezerjacobs** [cloudscape-design/components#2521] [Q&A] [answered]
I can't say that I know what I'm talking about but does it make sense to apply [this](https://css-tricks.com/you-want-minmax10px-1fr-not-1fr/) or any of the other "grid blowout" solutions to src/column-layout/flexible-column-layout/index.tsx ?
I noticed some long values in the last column were overflowing and I think this would fix it.

**Comment by gethinwebster** [comment, marked as answer]
Yes, it would make sense, in fact we're already working on this! Hopefully a PR should be coming soon.
